.travis.sh

```diff
@@ -22,6 +22,5 @@ cd /var/source
 hhvm -d hhvm.jit=0 /usr/local/bin/composer install
 
 hh_server --check $(pwd)
-hhvm -d hhvm.php7.all=0 -d hhvm.jit=0 vendor/bin/phpunit
-hhvm -d hhvm.php7.all=1 -d hhvm.jit=0 vendor/bin/phpunit
-hhvm -d hhvm.php7.all=1 -d hhvm.jit=1 vendor/bin/phpunit
+hhvm vendor/bin/phpunit
+hhvm vendor/bin/hhast-lint
```

Comment:
simplified here while I was at it as in 3.26+ php7 mode is the only supported mode